common/utils/src/main/resources/error/error-conditions.json

```
@@ -1091,6 +1091,12 @@
     ],
     "sqlState" : "KD010"
   },
+  "DATA_SOURCE_UNSUPPORTED_DML_OPERATION" : {
+    "message" : [
+      "Data source '<provider>' does not support requested DML operation: <sqlOperation>"
```

```suggestion
      "Data source '<provider>' does not support requested operation: <sqlOperation>"
```
We can be more generic.

---

Done